pytorch_lightning/callbacks/device_stats_monitor.py

```diff
@@ -65,7 +65,9 @@ def on_train_batch_start(
         device_stats = trainer.accelerator.get_device_stats(device)
         for logger in trainer.loggers:
             separator = logger.group_separator
-            prefixed_device_stats = _prefix_metric_keys(device_stats, "on_train_batch_start", separator)
+            prefixed_device_stats = _prefix_metric_keys(
```

Comment:
I think adding this is fine because the device stats are already prefixed with the hook names, however, I wouldn't do this for the other callbacks as modified in this PR